src/metabase/public_settings.clj

```diff
@@ -114,18 +114,32 @@
   :audit      :getter)
 
   ;; TODO: the settings on the BE will be done at the end, with help from the backend team
-  (defsetting help-link
+(defsetting help-link
   (deferred-tru "TODO")
   :default    :metabase_default
   :type       :keyword
   :audit      :getter
   :visibility :public)
 
+(defn- normalize-help-link-custom-destination [^String s]
```

Comment:
This is wip code just to receive an error, the error will actually be handled in the next PR.
Real BE validation should come later with help from the webapp admin team